README.md

```diff
@@ -51,10 +51,17 @@ C:\php-sdk\php-<version>\vc15\x64\php-src
 $ nmake test-awscrt
 ```
 
+Note: for VS2017, Cmake will default to build for Win32, refer to [here](https://cmake.org/cmake/help/latest/generator/Visual%20Studio%2015%202017.html). If you are building for x64 php, you can set environment variable as follow to let cmake pick x64 compiler.
+
+```bat
+set CMAKE_GENERATOR=Visual Studio 15 2017
+set CMAKE_GENERATOR_PLATFORM=x64
+```
+
```

Comment:
I never needed to do this on my machine

I believe using the recommended scripts like `phpsdk-vs16-x64.bat` from [php-sdk-binary-tools](https://github.com/microsoft/php-sdk-binary-tools) sets this up to work correctly...

Maybe just revert changes to the README we'll fix it up later once we get a better grip on all this?

Reply:
from the customer issue [here](https://github.com/awslabs/aws-crt-php/issues/68) and what I tested with the instruction from our README. 
```
""" From VS2017 Command Prompt
> C:\php-sdk\phpsdk-vc15-x64.bat
```

I met the issue that Cmake picked up x86 as generator platform. You can check the log from the customer issue. The `phpsdk-vc15-x64.bat` doesn't set the target platform correctly I believe.